rtos/ThisThread.h

```diff
@@ -174,11 +174,17 @@ void sleep_until(uint64_t millisec);
 void yield();
 
 /** Get the thread id of the current running thread.
-  @return  thread ID for reference by other functions or NULL in case of error or in ISR context.
+  @return  thread ID for reference by other functions or NULL in case of error.
   @note You may call this function from ISR context.
 */
 osThreadId_t get_id();
 
+/** Get the thread name of the current running thread.
+  @return  thread name pointer or NULL in case of error or in ISR context.
+  @note You cannot call this function from ISR context.
+*/
+const char *get_name();
```

Comment:
If it's documented as returning NULL in ISR context, then clearly you can call it from ISR context. Not consistent.

Reading CMIS-RTOS docs, they don't _explicitly_ say that `osThreadGetId` returns `NULL` if called from ISR, so that you know that you're not making an illegal call to `osThreadGetName` from ISR.

And looking at the RTX implementation, I think it does actually does return the current foreground thread - presumably in case you're trying profiling?  So from ISR it will probably crash due to an error callback from the `osThreadGetName`

So "not callable from ISR" is correct, so remove the "NULL in ISR context" comment.